### ISSUE TITLE
[Bug]: Updates to existing CO records not reflected in Filter

### ISSUE THREAD

**Issue by AntWeb-org** [SpeciesFileGroup/taxonworks#3708]
### Steps to reproduce the bug

```bash
Objective: reviewing newly created CO records AND updated (existing) CO records for 12/12/2023
I made updates to existing CO records from within the CSD task: OTUs, determiner and det. dates.
I also created new CO records in the CSD task.
I wanted to review my entries and updates for the day.
Using the CO Filter, I selected today's date, "Both" for "created at" and "updated at" and my name under Housekeeping for a list of COs.
The results yielded only the newly created CO records for the day.
Selecting only "updated at", the results yielded the same newly created CO records.
Reviewing Recent Updates from Data/Collecting Objects, the updated COs are not on the list with the newly created COs. 
I made updates to existing COs after the newly created COs.
See screenshots
```

### Screenshot

A portion of my CO Filter selections alongside results of only the newly created CO records:

![Untitled-1](https://github.com/SpeciesFileGroup/taxonworks/assets/74991451/71003d2b-d063-48a1-bf91-8265ac138992)

From Data/Collecting Objects, the updated COs are not listed (I updated them after creating new COs)

![Untitled-2](https://github.com/SpeciesFileGroup/taxonworks/assets/74991451/a0f4dd96-5cf9-4e89-944f-584feabb4c80)

Update made for collection_object_id=1669549:

![image](https://github.com/SpeciesFileGroup/taxonworks/assets/74991451/16dd58bb-5c36-4ca7-ba67-6c82ea1be03a)

Update noted under Timeline:

![image](https://github.com/SpeciesFileGroup/taxonworks/assets/74991451/07e5d343-7808-4421-b16f-31fa1c1876d5)

### Expected behavior

The filters should produce results for all updated COs (and CEs) from Housekeeping based when "updated at" is selected. 

### Environment

Production

### Version

 AntWeb v0.36.0

### Browser Used

Chrome Version 120.0.6099.71

**Comment by mjy** [comment]
You are correct, adding TaxonDeterminations does not trigger the Housekeeping on CollectionObjects (only edits on the objects properties as they are seen in the table do this).  I.e. there is a different housekeeping updated/created on the TaxonDeterminiation, DataAttributes etc.

We'll have to add an option to expand the filter to include checking these related values. Perhaps this is done in part conjunction with the Activity feed concept.

**Comment by AntWeb-org** [comment]
fixed
